lib/ReactViews/FeatureInfo/FeatureInfoPanel.jsx

```diff
@@ -200,8 +204,9 @@ function determineCatalogItem(nowViewing, feature) {
         }
 
         for (i = nowViewing.items.length - 1; i >= 0; i--) {
-            if (nowViewing.items[i].dataSource === dataSource) {
-                result = nowViewing.items[i];
+            item = nowViewing.items[i];
+            if (item.dataSource === dataSource || (defined(item._geoJsonItem) && item._geoJsonItem.dataSource === dataSource)) {
```

Comment:
That's a great pickup - actually the WPS is special, it's the only catalog item that maps it across - but you're right, I think a much better way to effect this would be to expose `_geoJsonItem.dataSource` as the `dataSource` for all the other ones too. I was feeling uncomfortable referring to an underscored property outside the class!